syft/pkg/cataloger/redhat/parse_rpm_manifest.go

```diff
@@ -44,7 +44,10 @@ func parseRpmManifest(_ context.Context, _ file.Resolver, _ *generic.Environment
 			continue
 		}
 
-		p := newDBPackage(reader.Location, *metadata, nil, nil)
+		// Create a new package representing the RPM manifest entry
+		// note: we don't know the quality of the manifest dependency information, so even though there are multiple
```

Comment:
Can you help me understand why this is "Incomplete" and not "Unknown"? It sounds like we don't know the completeness state, which sounds "unknown" to me.

How is this different than the elf metadata cataloger, where we say, "someone could link extra stuff in at runtime and we'll never know, so unknown"?

Reply:
This was changed to "unknown" per https://github.com/anchore/syft/pull/3402#discussion_r1871423812